docs/kyma-environment-broker/03-01-service-description.md

```diff
@@ -69,7 +91,65 @@ These are the provisioning parameters for Azure that you can configure:
 | **autoScalerMax** | int | Specifies the maximum number of virtual machines to create. | No | `4` |
 | **maxSurge** | int | Specifies the maximum number of virtual machines that are created during an update. | No | `4` |
 | **maxUnavailable** | int | Specifies the maximum number of VMs that can be unavailable during an update. | No | `1` |
-| **providerSpecificConfig.AzureConfig.VnetCidr** | string | Provides configuration variables specific for Azure. | No | `10.250.0.0/19` |
+
+ </details>
+
+  <details>
+  <summary label="azure-lite-plan">
+  Azure HA
+  </summary>
+
+| Parameter name | Type | Description | Required | Default value |
+| ---------------|-------|-------------|:----------:|---------------|
+| **machineType** | string | Specifies the provider-specific virtual machine type. | No | `Standard_D4_v3` |
```

Comment:
?! there is no such machine type technical name in AWS, machineTypes: []string{"m5.2xlarge", "m5.4xlarge", "m5.8xlarge", "m5.12xlarge", "m4.2xlarge", "m4.4xlarge", "m4.10xlarge", "m4.16xlarge"},

Reply:
copy-paste typo :(